border0/provider.go

```diff
@@ -20,6 +20,8 @@ const (
 	maxParallelism = 10
 
 	defaultTimeout = time.Second * 30
+
+	replicationDelay = time.Second
```

Comment:
how about `defaultReplicationLagDelay`, and make it overridable like the http client timeout?